Doppler.Sap/Controllers/TestSapController.cs

```diff
@@ -0,0 +1,23 @@
+using System.Threading.Tasks;
+using Doppler.Sap.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Doppler.Sap.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class TestSapController
+    {
+        private readonly ITestSapService _testSapService;
+
+        public TestSapController(ITestSapService testSapService) => _testSapService = testSapService;
+
+        [HttpGet("/testSap")]
```

Comment:
What do you think about

```suggestion
        [HttpGet("/test-sap-connection")]
```